services/shhext/api.go

```diff
@@ -574,10 +520,13 @@ func (api *PublicAPI) processPFSMessage(msg *whisper.Message) error {
 		handler := EnvelopeSignalHandler{}
 		handler.DecryptMessageFailed(keyString)
 		return nil
+	} else if err == chat.ErrNotProtocolMessage {
```

Comment:
Maybe this whole comparison block should be converted to a `switch {}` block?

Reply:
done!